### PR TITLE
Add async commit fields to LockInfo

### PR DESCRIPTION
**Pull request by MyonKeminta**
This PR added fields about parallel commit to `LockInfo`.

### CODE REVIEW

File: proto/kvrpcpb.proto [hunk 1]
```diff
@@ -711,6 +711,10 @@ message LockInfo {
     uint64 txn_size = 5;
     Op lock_type = 6;
     uint64 lock_for_update_ts = 7;
+    // Fields for transactions that are using Parallel Commit.
+    bool use_parallel_commit = 8;
```
Comment (youjiali1995) [inline proto/kvrpcpb.proto:715]: How about renaming it to async_commit?

Comment (sticnarf) [reply proto/kvrpcpb.proto:715]: Oh, we should rename all occurances of "parallel commit" to "async commit"?

Comment (youjiali1995) [reply proto/kvrpcpb.proto:715]: Yes.

Comment (MyonKeminta) [reply proto/kvrpcpb.proto:715]: Ok. Let me do this in this PR.

**Comment by MyonKeminta** [comment]
@youjiali1995 PTAL again thanks!

**Pull request merged**